docs/content/en/docs/yaml-crd-ref/taskdefinition.md

```diff
@@ -3,3 +3,310 @@ title: KeptnTaskDefinition
 description: Define tasks that can be run pre- or post-deployment
 weight: 89
 ---
+
+
+A `KeptnTaskDefinition` defines tasks
+that are run by the Keptn Lifecycle Toolkit
+as part of the pre- and post-deployment phases of a
+[KeptnApp](./app.md).
```

Comment:
```suggestion
[KeptnApp](./app.md) or [KeptnWorkload](./workload.md).
```

Reply:
@thisthat We do not have a ref page for the KeptnWorkload yaml file because I think it is autogenerated and users do not need to populate it.  Do we want a KeptnWorkload page in the yaml-crd-ref directory or shall I link to [Keptn automatic app discovery](../implementing/integrate.md/#use-keptn-automatic-app-discovery) ?

Reply:
For the time being, I added a link to the "Workloads" page in "Concepts".    Probably not the most perfect solution for the long run but we need to revisit all sorts of structural issues in the docs after we get these Getting Started guides published.